docker-base/Dockerfile

```diff
@@ -29,47 +28,38 @@ RUN apt-get update -qq \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
 
+ARG PETSC_CONFIG_OPTIONS="--prefix=/petsc-3.5.4 --with-debugging=0 --with-shared-libraries --download-mumps=1 --download-fblaslapack=1 --download-parmetis=1 --download-metis=1 --download-scalapack=1 --download-blacs=1 --download-superlu_dist=1 --with-mpi=1"
+
 RUN wget http://ftp.mcs.anl.gov/pub/petsc/release-snapshots/petsc-3.5.4.tar.gz \
     && tar -xvf petsc-3.5.4.tar.gz \
     && mv petsc-3.5.4 petsc-3.5.4-src \
     && cd petsc-3.5.4-src \
-    && PETSC_DIR=$PWD ./config/configure.py --prefix=/petsc-3.5.4 --with-debugging=0 --with-shared-libraries --download-mumps=1 --download-fblaslapack=1 --download-parmetis=1 --download-metis=1 --download-scalapack=1 --download-blacs=1 --download-superlu_dist=1 --with-mpi=1 \
+    && PETSC_DIR=$PWD ./config/configure.py ${PETSC_CONFIG_OPTIONS} \
     && make PETSC_DIR=/petsc-3.5.4-src PETSC_ARCH=arch-linux2-c-opt all \
     && make PETSC_DIR=/petsc-3.5.4-src PETSC_ARCH=arch-linux2-c-opt install \
     && rm -rf petsc-3.5.4.tar.gz \
     && rm -rf petsc-3.5.4-src
 
-RUN git clone --depth=1 https://github.com/LukeMondy/EarthByte_Underworld.git  \
-    && cd /EarthByte_Underworld \
-    && ./configure.py \
-        --hdf5-lib-dir=/usr/lib/x86_64-linux-gnu/hdf5/serial \
+ARG CONFIG_OPTIONS="--hdf5-lib-dir=/usr/lib/x86_64-linux-gnu/hdf5/serial \
         --hdf5-inc-dir=/usr/include/hdf5/serial \
         --petsc-dir=/petsc-3.5.4 \
         --with-debugging=0 \
-        --cflags="-Wno-error=format-security" \
+        --cflags='-Wno-error=format-security'"
+
+RUN git clone --depth=1 https://github.com/rbeucher/EarthByte_Underworld.git  \
```

Comment:
I would prefer that this git link points to whatever repo the Docker file is in - maybe we can have the URL as an ARG with the default as this repo (or if it moves, whatever repo that is).

Reply:
Oh I did not see it was pointing to my fork...

Reply:
I have added an `UW_EARTHBYTE` ARG. What do you think?

Reply:
Maybe `UW_GIT_URL`? No need to get more Earthbyte in there than necessary.